PaolaSantolloV/react-apprenticeship-capstone-2
# Challenge 2

## Description
This PR contain the capstone project 2

- Add husky, prettier and eslint on the project
- Add hooks for the request
- Add the functional components
- Add test for responsive
- Add test for hooks and functional components
- Add styles on the components

## Type of Change

Capstone project 2



## How should this be tested?

In your terminal run `npm install `after that `npm run start`, now you can see the page on [localhost:3000]( http://localhost:3000)
To run the test `npm run test`, if you want to watch the coverage `npm run test a -- --coverage`.

## What is the deliverable?

Final deliverable

## Screenshots and links
netlify link: [ https://capstone2.netlify.app/]( https://adoring-galileo-25c222.netlify.app/)


## Checklist
- [ x] I added tests to prove that my fix is effective or my deliverable works.
